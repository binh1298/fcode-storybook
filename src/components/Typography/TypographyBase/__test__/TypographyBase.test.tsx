import TyphographyBase, { TypographyBaseProps } from "..";

import { render, RenderResult, screen } from "@testing-library/react";

describe("<TypographyBase />", () => {
    let wrapper: RenderResult;

    const props: TypographyBaseProps = {
        children: "Test",
    };

    beforeEach(() => {
        wrapper = render(<TyphographyBase {...props} />);
    });

    it("should exist children", () => {
        expect(screen.getByText("Test")).toBeInTheDocument();
    });

    it("should match snapshot", () => {
        expect(wrapper.container).toMatchSnapshot();
    });
});
