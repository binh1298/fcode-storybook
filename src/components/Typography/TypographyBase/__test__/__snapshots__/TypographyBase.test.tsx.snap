// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TypographyBase /> should match snapshot 1`] = `
<div>
  <p
    class="MuiTypography-root MuiTypography-body1"
  >
    Test
  </p>
</div>
`;
